src/towncrier/newsfragments/603.feature.rst

```diff
@@ -0,0 +1,4 @@
+The ``towncrier create`` action now uses sections defined in your config (either interactively, or via the new ``--section`` option).
+
+If you use sections and none have an empty path, you must now specify the section when creating a new news fragment.
```

Comment:
this looks like it's breacking backward compatibility ?

Can we continue to have the old behaviour for the case when `section` is not specified?

--------

If this change is needed, then this text should not be placed in a `.feature` fragment

Instead, it should be in an `.removal` to be highlighted as part of "breaking changes"

Reply:
My reasoning behind it **not** being backwards incompatible is that if you didn't have any empty-path sections, then `create` wouldn't have been working for you in the past anyway

I'd be fine moving this to removal if it's actually considered backwards incompatible. Or maybe the second paragraph is just confusing and redundant since it was behaviour that didn't work previously anyway.

Reply:
If it was not working before, I think that we can just remove this info.
As part of #606 we can find document the default section.

Thanks

Reply:
Removed, I've gone with the concept of a default section and this isn't necessary to say here.